docs/reference-guides/block-api/block-registration.md

```diff
@@ -253,6 +253,18 @@ Blocks are able to be inserted into blocks that use [`InnerBlocks`](https://gith
 
 Setting `parent` lets a block require that it is only available when nested within the specified blocks.
 
+```js
+// Only allow this block when it is nested in a Columns block
+parent: [ 'core/columns' ],
+```
+
+#### ancestor (optional)
+
+-   **Type:** `Array`
+
+The `ancestor` property makes a block available inside the specified block types at any position of the ancestor block subtree. That allows, for example, to place a ‘Comment Content’ block inside a ‘Column’ block, as long as ‘Column’ is somewhere within a ‘Comment Template’ block. In comparrison to the `parent` property blocks that specify their `ancestor` can be placed anywhere in the subtree whilst blocks with a specified `parent` need to be direct children.
+
+
 ```js
 // Only allow this block when it is nested in a Columns block
 parent: [ 'core/columns' ],
```

Comment:
This should be ancestor, not parent; since the code example that you added at line 256 is for the parent attribute; not the ancestor attribute.

Reply:
Fixed.